### PR TITLE
string-manipulation-3 - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**
string-manipulation-3 - 1

### CODE REVIEW

File: string-manipulation-3/string-manipulation-3/src/title-case.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/* exported titleCase */
+function titleCase(title) {
+  const titleArray = title.toLowerCase().split(' ');
+  console.log(titleArray);
```
Comment (shawnkost) [inline string-manipulation-3/string-manipulation-3/src/title-case.js]: `console.log` is only for debugging / testing purposes, so they should be removed whenever submitting an exercise! Unless we specifically ask for you to include them

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just need to remove a console log here Roland!

File: string-manipulation-3/string-manipulation-3/src/title-case.js (hunk 1 above)
Comment (restropiajr) [reply string-manipulation-3/string-manipulation-3/src/title-case.js]: Sorry forgot to remove this when I was debugging.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**